pkg/configtx/msp.go

```diff
@@ -71,8 +71,8 @@ type MSP struct {
 	// List of TLS intermediate certificates trusted by this MSP;
 	// They are returned by GetTLSIntermediateCerts.
 	TLSIntermediateCerts []*x509.Certificate
-	// fabric_node_ous contains the configuration to distinguish clients from peers from orderers
-	// based on the OUs.
+	// fabric_node_ous contains the configuration to distinguish clients
```

Comment:
it's not fabric_node_ous, can you check the other comments to make sure they match with the field theyre describing?

Reply:
🧹